src/backend/BSSched/BuildJob.pm

```diff
@@ -544,11 +544,12 @@ sub jobfinished {
     }
     # Add a comment to logfile from last real build
     BSUtil::appendstr("$dst/logfile", "\nRetried build at ".localtime(time())." returned same result, skipped\n");
+    my $jobhist = makejobhist($info, $status, $js, 'unchanged');
+    addbuildstats($jobdatadir, $dst, $jobhist) if ($all{'_statistics'});
```

Comment:
the parens are not needed if the `if` is a statement modifier. Can you please remove them so that the code fits the style of the rest of the build service?

I.e. just use `addbuildstats($jobdatadir, $dst, $jobhist) if $all{'_statistics'};`

Same for the other `if` modifiers in the rest of the commit.